### ISSUE TITLE
Publish version 1.1.0

### ISSUE THREAD

**Issue by rogierschouten** [sinonjs/nise#9]
I wanted to report the same issue as #5 but I found out it is already fixed. Could you please do NPM publish of 1.0.2?

**Comment by fatso83** [comment]
sure, will do asap

**Comment by fatso83** [comment]
@mroderick Can you give me publish rights on NPM? Or add me to the right organization or something (maybe we should namescope the packages under `sinon/` - nah, different discussion)?

**Comment by rogierschouten** [comment]
ping

**Comment by mroderick** [comment]
I've added everyone in the @sinonjs/core group

```text
$ npm owner add fatso83
+ fatso83 (nise)

$ npm owner add mantoni
+ mantoni (nise)

$ npm owner add cjohansen
+ cjohansen (nise)
```

**Comment by fatso83** [comment]
Published 1.1.0 now on npm.
